auth/src/main/java/com/firebase/ui/auth/ui/FragmentHelper.java

```diff
@@ -30,7 +31,9 @@ public void finish(int resultCode, Intent intent) {
     @Override
     public void showLoadingDialog(String message) {
         dismissDialog();
-        mProgressDialog = ProgressDialog.show(mFragment.getContext(), "", message, true);
+        ContextThemeWrapper context =
+                new ContextThemeWrapper(mFragment.getContext(), getFlowParams().themeId);
```

Comment:
We shouldn't need to check for `0` because we do this: `private int mTheme = getDefaultTheme();`.